### PR TITLE
Adding README

### PR DESCRIPTION
**Pull request by vncoelho**
@lightszero, @igormcoelho, @shargon, @erikzhang, can you take a look?

@erikzhang, this project is under travis.com ( I think that neo-vm  is already on .com as well), while neo-cli  and neo are under travis.org

Maybe switch the other to .com as well?

### CODE REVIEW

**Comment by vncoelho** [comment]
@erikzhang,

Maybe just remove the .org from all neo-projects:

image

And leave the travis.com

I read about this some time ago, but I forgot the history...

**Comment by vncoelho** [comment]
image

https://travis-ci.org/organizations/neo-project/repositories

**Comment by vncoelho** [comment]
https://devops.stackexchange.com/questions/1201/whats-the-difference-between-travis-ci-org-and-travis-ci-com

image

**Comment by erikzhang** [comment]
There are currently no options for manual migration, we can only wait for their automatic migration plan.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  </a>
+  <a href="https://github.com/neo-project/neo-devpack-dotnet/blob/master/LICENSE">
+    <img src="https://img.shields.io/badge/license-MIT-blue.svg">
+  </a>
+
+</p>
+
+# NEO Devpack Dotnet
+
+This is the set of development tools for Dotnet.
```
Comment (erikzhang) [inline README.md]:
```suggestion
This is the set of development tools for .NET.
```

**Comment by vncoelho** [comment]
@erikzhang, you can just disable that two on the buttons I highlighted on the printscreen.
After that, you to integrations and enable travis.com
I needed to do this for a project we were doing.
Both were running in parallel. I think that this might also be the case.

Otherwise, just send an email to travis support. I think it is worth, it will improve our experience.

**Pull request merged**